composer.json

```diff
@@ -35,7 +37,19 @@
     },
     "config": {
```

Comment:
Not a requirement, rather a nice-to-have, but for consistency across all our packages, let's add under the `config` section:
`"sort-packages": true,`